boards/posix/native_posix/doc/board.rst

```diff
@@ -168,6 +168,23 @@ available options::
      [-no-rt]                  :Do NOT slow down the execution to real time, but
                                 advance Zephyr's time as fast as possible and
                                 decoupled from the host time
+     [-rt-drift=<drift>]       :In parts per 1, drift of the simulated clock
+                                relative to the real host time.
+                                Normally this would be set to a value of a few
+                                ppm (e.g. 50e-6)
+                                This option has no effect in non-real time mode
+     [-rt-radio=<dratio>]      :Relative speed of the simulated time vs real
+                                time. For ex. set to 2 to have simulated time
```

Comment:
a bit clearer if this line read:

    time, for example, set to 2 to have simulated time